src/app/services/users-role-helper/users-role-helper.coffee

```diff
@@ -0,0 +1,29 @@
+@App.service 'UsersRole', (USER_ROLES, MnoeUsers, toastr, $translate) ->
+  _self = @
+
+  @availableRoles = _.map(USER_ROLES, 'value')
+
+  @keyFromRole = (role) ->
+    _.find(USER_ROLES, 'value', role).label
+
+  @updateUserRole = (organization, user, oldRole) ->
```

Comment:
I think this should be done at the controller level as it has a few things that could be specific to each component (ie. error handling, attrs that affect the view).